app/main.ipynb

```diff
@@ -446,30 +1617,70 @@
     "    return result\n",
     "\n",
     "matrix1 = [[1, 2], [3, 4]]\n",
-    "matrix2 = [[5, 6], [7, 8]]"
+    "matrix2 = [[5, 6], [7, 8]]\n",
+    "%time unknown_signature(matrix1, matrix2)"
```

Comment:
The function `unknown_signature` correctly performs element-wise multiplication of two matrices using nested loops.